lib/WebSocket.js

```diff
@@ -262,6 +265,14 @@ WebSocket.prototype.send = function send(data, options, cb) {
 
       if (typeof cb === 'function') cb(error);
     });
+  } else if (Array.isArray(data)) {
+    // send array as iovecs
+    // using fin just like sending a stream but there's no need to
+    // wait for the stream data to be ready, so push all to the sender.
+    for (var i = 0; i < data.length; ++i) {
+      options.fin = (i + 1 === data.length);
+      this._sender.send(data[i], options, options.fin ? cb : undefined);
```

Comment:
I was really going by function logic here, not so much use-case. But an array of strings... unthinkable?